### ISSUE TITLE
Favorites menu for categories broken 

### ISSUE THREAD

**Issue by coreybruce** [jellyfin/jellyfin#12802]
### This issue respects the following points:

- [X] This is a **bug**, not a question or a configuration issue; Please visit our forum or chat rooms first to troubleshoot with volunteers, before creating a report. The links can be found [here](https://jellyfin.org/contact/).
- [X] This issue is **not** already reported on [GitHub](https://github.com/jellyfin/jellyfin/issues?q=is%3Aopen+is%3Aissue) _(I've searched it)_.
- [X] I'm using an up to date version of Jellyfin Server stable, unstable or master; We generally do not support previous older versions. If possible, please update to the latest version before opening an issue.
- [X] I agree to follow Jellyfin's [Code of Conduct](https://jellyfin.org/docs/general/community-standards.html#code-of-conduct).
- [X] This report addresses only a single issue; If you encounter multiple issues, kindly create separate reports for each one.

### Description of the bug

After upgrading to Jellyfin-server 10.9.11-1 if you try and enter the favorites menu for any category it  is broken and will be stuck loading, I looked at the console in firefox and see a javascript error.

### Reproduction steps

1. sign in
2. go to favorites 
3. select category in your favorites

### What is the current _bug_ behavior?

It will be infinity loading

### What is the expected _correct_ behavior?

Load the favorites category menu

### Jellyfin Server version

10.9.11+

### Specify version number

10.9.11-1

### Specify the build version

10.9.11

### Environment

```markdown
- OS: Manjaro
- Linux Kernel: 6.6.54-2-MANJARO
- Virtualization: no
- Clients: Jellyfin-Media-Player
- Browser: Firefox
- FFmpeg Version: 1:6.0.1p8-1
- Playback Method: 
- Hardware Acceleration: yes
- GPU Model: Nvidia GTX 1060 6gb
- Plugins:none
- Reverse Proxy:
- Base URL: none
- Networking: Host
- Storage: Local
```

### Jellyfin logs

```shell
Logs don't show the issue
```

### Client / Browser logs

```
Uncaught (in promise) TypeError: e is null
    e http://corey-serverpc.xerus-tuna.ts.net:8096/web/list.ea9e763cb335ac6a57c1.chunk.js:2
    promise callback*23416/e/< http://corey-serverpc.xerus-tuna.ts.net:8096/web/list.ea9e763cb335ac6a57c1.chunk.js:2
    d http://corey-serverpc.xerus-tuna.ts.net:8096/web/main.jellyfin.bundle.js?dec8fb79ce2a75e4a201:2
    value http://corey-serverpc.xerus-tuna.ts.net:8096/web/main.jellyfin.bundle.js?dec8fb79ce2a75e4a201:2
    promise callback*value http://corey-serverpc.xerus-tuna.ts.net:8096/web/main.jellyfin.bundle.js?dec8fb79ce2a75e4a201:2
    f http://corey-serverpc.xerus-tuna.ts.net:8096/web/main.jellyfin.bundle.js?dec8fb79ce2a75e4a201:2
    s http://corey-serverpc.xerus-tuna.ts.net:8096/web/main.jellyfin.bundle.js?dec8fb79ce2a75e4a201:2
    s http://corey-serverpc.xerus-tuna.ts.net:8096/web/main.jellyfin.bundle.js?dec8fb79ce2a75e4a201:2
    a http://corey-serverpc.xerus-tuna.ts.net:8096/web/main.jellyfin.bundle.js?dec8fb79ce2a75e4a201:2
    promise callback*l http://corey-serverpc.xerus-tuna.ts.net:8096/web/main.jellyfin.bundle.js?dec8fb79ce2a75e4a201:2
    u http://corey-serverpc.xerus-tuna.ts.net:8096/web/main.jellyfin.bundle.js?dec8fb79ce2a75e4a201:2
    u http://corey-serverpc.xerus-tuna.ts.net:8096/web/main.jellyfin.bundle.js?dec8fb79ce2a75e4a201:2
    f http://corey-serverpc.xerus-tuna.ts.net:8096/web/main.jellyfin.bundle.js?dec8fb79ce2a75e4a201:2
    A http://corey-serverpc.xerus-tuna.ts.net:8096/web/main.jellyfin.bundle.js?dec8fb79ce2a75e4a201:2
    s http://corey-serverpc.xerus-tuna.ts.net:8096/web/main.jellyfin.bundle.js?dec8fb79ce2a75e4a201:2
    s http://corey-serverpc.xerus-tuna.ts.net:8096/web/main.jellyfin.bundle.js?dec8fb79ce2a75e4a201:2
    u http://corey-serverpc.xerus-tuna.ts.net:8096/web/main.jellyfin.bundle.js?dec8fb79ce2a75e4a201:2
    u http://corey-serverpc.xerus-tuna.ts.net:8096/web/main.jellyfin.bundle.js?dec8fb79ce2a75e4a201:2
    A http://corey-serverpc.xerus-tuna.ts.net:8096/web/main.jellyfin.bundle.js?dec8fb79ce2a75e4a201:2
    promise callback*12854/t.A/< http://corey-serverpc.xerus-tuna.ts.net:8096/web/main.jellyfin.bundle.js?dec8fb79ce2a75e4a201:2
    Fi http://corey-serverpc.xerus-tuna.ts.net:8096/web/node_modules.react-dom.bundle.js?dec8fb79ce2a75e4a201:2
    unstable_runWithPriority http://corey-serverpc.xerus-tuna.ts.net:8096/web/60232.bundle.js?dec8fb79ce2a75e4a201:2
    jl http://corey-serverpc.xerus-tuna.ts.net:8096/web/node_modules.react-dom.bundle.js?dec8fb79ce2a75e4a201:2
    Mi http://corey-serverpc.xerus-tuna.ts.net:8096/web/node_modules.react-dom.bundle.js?dec8fb79ce2a75e4a201:2
    vi http://corey-serverpc.xerus-tuna.ts.net:8096/web/node_modules.react-dom.bundle.js?dec8fb79ce2a75e4a201:2
    ql http://corey-serverpc.xerus-tuna.ts.net:8096/web/node_modules.react-dom.bundle.js?dec8fb79ce2a75e4a201:2
    unstable_runWithPriority http://corey-serverpc.xerus-tuna.ts.net:8096/web/60232.bundle.js?dec8fb79ce2a75e4a201:2
    jl http://corey-serverpc.xerus-tuna.ts.net:8096/web/node_modules.react-dom.bundle.js?dec8fb79ce2a75e4a201:2
    ql http://corey-serverpc.xerus-tuna.ts.net:8096/web/node_modules.react-dom.bundle.js?dec8fb79ce2a75e4a201:2
    Kl http://corey-serverpc.xerus-tuna.ts.net:8096/web/node_modules.react-dom.bundle.js?dec8fb79ce2a75e4a201:2
    di http://corey-serverpc.xerus-tuna.ts.net:8096/web/node_modules.react-dom.bundle.js?dec8fb79ce2a75e4a201:2
    Lo http://corey-serverpc.xerus-tuna.ts.net:8096/web/node_modules.react-dom.bundle.js?dec8fb79ce2a75e4a201:2
    t http://corey-serverpc.xerus-tuna.ts.net:8096/web/main.jellyfin.bundle.js?dec8fb79ce2a75e4a201:2
    d http://corey-serverpc.xerus-tuna.ts.net:8096/web/main.jellyfin.bundle.js?dec8fb79ce2a75e4a201:2
    promise callback*14847/d/< http://corey-serverpc.xerus-tuna.ts.net:8096/web/main.jellyfin.bundle.js?dec8fb79ce2a75e4a201:2
    Fi http://corey-serverpc.xerus-tuna.ts.net:8096/web/node_modules.react-dom.bundle.js?dec8fb79ce2a75e4a201:2
    unstable_runWithPriority http://corey-serverpc.xerus-tuna.ts.net:8096/web/60232.bundle.js?dec8fb79ce2a75e4a201:2
    jl http://corey-serverpc.xerus-tuna.ts.net:8096/web/node_modules.react-dom.bundle.js?dec8fb79ce2a75e4a201:2
    Mi http://corey-serverpc.xerus-tuna.ts.net:8096/web/node_modules.react-dom.bundle.js?dec8fb79ce2a75e4a201:2
    Oi http://corey-serverpc.xerus-tuna.ts.net:8096/web/node_modules.react-dom.bundle.js?dec8fb79ce2a75e4a201:2
    N http://corey-serverpc.xerus-tuna.ts.net:8096/web/60232.bundle.js?dec8fb79ce2a75e4a201:2
    onmessage http://corey-serverpc.xerus-tuna.ts.net:8096/web/60232.bundle.js?dec8fb79ce2a75e4a201:2
    77699 http://corey-serverpc.xerus-tuna.ts.net:8096/web/60232.bundle.js?dec8fb79ce2a75e4a201:2
    s http://corey-serverpc.xerus-tuna.ts.net:8096/web/runtime.bundle.js?dec8fb79ce2a75e4a201:1
    91434 http://corey-serverpc.xerus-tuna.ts.net:8096/web/60232.bundle.js?dec8fb79ce2a75e4a201:2
    s http://corey-serverpc.xerus-tuna.ts.net:8096/web/runtime.bundle.js?dec8fb79ce2a75e4a201:1
    76067 http://corey-serverpc.xerus-tuna.ts.net:8096/web/node_modules.react-dom.bundle.js?dec8fb79ce2a75e4a201:2
    s http://corey-serverpc.xerus-tuna.ts.net:8096/web/runtime.bundle.js?dec8fb79ce2a75e4a201:1
    78325 http://corey-serverpc.xerus-tuna.ts.net:8096/web/node_modules.react-dom.bundle.js?dec8fb79ce2a75e4a201:2
    s http://corey-serverpc.xerus-tuna.ts.net:8096/web/runtime.bundle.js?dec8fb79ce2a75e4a201:1
    67816 http://corey-serverpc.xerus-tuna.ts.net:8096/web/main.jellyfin.bundle.js?dec8fb79ce2a75e4a201:2
    s http://corey-serverpc.xerus-tuna.ts.net:8096/web/runtime.bundle.js?dec8fb79ce2a75e4a201:1
    <anonymous> http://corey-serverpc.xerus-tuna.ts.net:8096/web/main.jellyfin.bundle.js?dec8fb79ce2a75e4a201:2
    O http://corey-serverpc.xerus-tuna.ts.net:8096/web/runtime.bundle.js?dec8fb79ce2a75e4a201:1
    <anonymous> http://corey-serverpc.xerus-tuna.ts.net:8096/web/main.jellyfin.bundle.js?dec8fb79ce2a75e4a201:2
    a http://corey-serverpc.xerus-tuna.ts.net:8096/web/runtime.bundle.js?dec8fb79ce2a75e4a201:1
    <anonymous> http://corey-serverpc.xerus-tuna.ts.net:8096/web/main.jellyfin.bundle.js?dec8fb79ce2a75e4a201:2
[list.ea9e763cb335ac6a57c1.chunk.js:2:30137](http://corey-serverpc.xerus-tuna.ts.net:8096/web/list.ea9e763cb335ac6a57c1.chunk.js)
```

### Relevant screenshots or videos

![image](https://github.com/user-attachments/assets/25b5bb97-c477-4221-81fd-472371a9d585)

**Comment by coreybruce** [comment]
> Please turn on debug logging, and see if the server produces any logs with debug logging enabled when you try to access the page

Hey there how do I enable debug logging?

**Comment by felix920506** [comment]
Please follow instructions here to enable debug logging
https://jellyfin.org/docs/general/administration/troubleshooting#debug-logging

**Comment by coreybruce** [comment]
> Please follow instructions here to enable debug logging https://jellyfin.org/docs/general/administration/troubleshooting#debug-logging

I hope I did it correctly and this log gives you the information that is needed

https://gist.github.com/coreybruce/90defc76fe9300c8da60a9941bd01629

**Comment by ronott** [comment]
got the same issue here. it will load the favourites if you change sorting or the filter (but never stop loading). what's missing are the play and shuffle links/buttons. which is really an issue with the favourite songs ;)

**Comment by ldexterldesign** [comment]
👋

Same

Holla if you need additional logs etc..?

Pretty annoying bug - I bet 99% of users will be using their favourites, especially for audio/music; like me

Regards

**Comment by crobibero** [comment]
I cannot reproduce the issue in latest unstable Jellyfin (10.10-2024101405)

**Comment by coreybruce** [comment]
> I cannot reproduce the issue in latest unstable Jellyfin (10.10-2024101405)

Right but we aren't using the latest unstable, were using the latest stable but can you confirm the issue on the latest stable? multiple people now are reporting the same issue an this issue shouldn't be closed yet.

**Comment by crobibero** [comment]
There will be no more 10.9 releases, so there is no point in keeping this open if it is already resolved for the next Jellyfin release. 

I did confirm the issue exists in 10.9.11, and is fixed in 10.10.

**Comment by coreybruce** [comment]
Alright thanks for letting us know!
